Jenkinsfile

```diff
@@ -495,8 +498,13 @@ pipeline {
                         stage("install") {
                             steps {
                                 buildName env.BUILD_NUMBER + "-" + BRANCH + "-" + env.GIT_COMMIT.substring(0, 7)
-                                sh "npm install --no-optional"
-                                sh "rm -rf ${GIT_CACHE_PATH}/*.lock"
+                                sh """
+                                    set -e
+                                    npm install --no-optional
```

Comment:
I think the `rm -rf` should be before the `npm install`

Reply:
Noticed after reviewing https://github.com/brave/brave-browser/pull/4802

Reply:
since you need rebase, I think you'll see it in the conflict - I can re-approve after that

Reply:
it doesn't matter, as long as it's before `init`

Reply:
have fixed now